packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -175,7 +175,7 @@ export class AppSyncSwiftVisitor extends AppSyncModelVisitor {
 
     return result.join('\n');
   }
-  protected getListType(typeStr: string): string {
+  protected getListType(typeStr: string, field: CodeGenField): string {
```

Comment:
Other ```getListType``` methods are returning a type that is either ```List<>``` or ```[]``` is it intentional here that just returning the incoming ```typeStr``` parameter? also no need for string templates as nothing is added to the incoming param.

Reply:
Yes this is intentional as the declaration block knows how to handle if its a list